src/Struct/InAppPurchaseCollection.php

```diff
@@ -26,4 +26,12 @@ public static function fromArray(array $data): self
 
         return new self($elements);
     }
+
+    /**
+     * @return array<int, string>
+     */
+    public function toArray(): array
```

Comment:
I think it's misleading to call this `toArray' when it actually returns a list of identifiers.